_i18n/en/_docs/c-api.md

```diff
@@ -22,6 +22,8 @@ See the [frida-core](https://github.com/frida/frida-core) repository for the sou
 
 frida-gum allows you to augment and replace functions using C.
 
+[This project](https://github.com/0xXA/snapchat-emulator-bypass) shows you how to augment `open`, `execve`, `__system_property_find`, and `__system_property_get` as well as how a patch can be applied from c only.
```

Comment:
I think we should have a section where we link to projects using Gum. However, this example no longer compiles with the latest Gum, and has some dead code (ExampleListener). I think we should focus these main docs on small examples that demonstrate the API.